### DISCUSSION TITLE
shorterall.com: detection

### DISCUSSION THREAD

**Discussion by yohanes4dip** [uBlockOrigin/uAssets#18449] [Q&A]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/124740436/235392297-c4fc6290-73a6-4a70-96a9-3f6a7683f21d.png), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable external filter lists, or I have verified that the issue still occurs without enabling external filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://shorterall.com/TzuQO`

### Category

detection

### Description

This page detect adblocker

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![image](https://github.com/uBlockOrigin/uAssets/assets/45124799/2552aba5-a809-4cc9-a571-e997e908dd00)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.49.2
Chromium: 114
filterset (summary): 
 network: 246445
 cosmetic: 140695
 scriptlet: 37936
 html: 0
listset (total-discarded, last-updated): 
 added: 
  adguard-annoyance: 67567-1720, 8h.55m
  adguard-generic: 71379-728, 8h.55m
  adguard-mobile: 8377-73, 8h.55m
  adguard-social: 20460-2011, 8h.55m
  adguard-spyware-url: 1053-3, 8h.55m
  block-lan: 47-0, 8h.55m
  curben-phishing: 111579-7, 8h.55m
  curben-pup: 199-0, 8h.55m
  dpollock-0: 11517-83, 8h.55m
  fanboy-thirdparty_social: 69-4, 8h.55m
  ublock-annoyances: 5092-223, 8h.55m
 default: 
  user-filters: 39-0, never
  IDN-0: 9510-3809, 8h.55m
  easylist: 67048-17855, 8h.55m
  easyprivacy: 31089-340, 8h.55m
  plowe-0: 3364-1221, 8h.55m
  ublock-abuse: 70-0, 8h.55m
  ublock-badware: 7257-393, 8h.55m
  ublock-filters: 33370-4292, 8h.55m
  ublock-privacy: 383-78, 8h.55m
  ublock-quick-fixes: 199-7, 8h.55m
  ublock-unbreak: 2033-99, 8h.55m
  urlhaus-1: 6644-3, 8h.55m
filterset (user): [array of 38 redacted]
trustedset: 
 added: [array of 81 redacted]
switchRuleset: 
 added: [array of 2 redacted]
modifiedUserSettings: 
 ignoreGenericCosmeticFilters: true
modifiedHiddenSettings: [none]
supportStats: 
 allReadyAfter: 1874 ms (selfie)
 maxAssetCacheWait: 1345 ms
popupPanel: 
 blocked: 12
 no-cosmetic-filtering: true
 network: 
  shorterall.com: 1
  adleadevent.com: 1
  cpx.to: 1
  criteo.com: 1
  diclotrans.com: 1
  googletagmanager.com: 1
  onetag-sys.com: 1
  quantcast.com: 1
  quantserve.com: 1
  sascdn.com: 1
  unblockia.com: 1
  zeotap.com: 1
```
</details>

**Comment by mapx-** [comment]
converted in discussion, who wants to contribute with filters for such a site is invited here.

**Reply by Lahiru2002** [reply]
I need this site filter.

**Comment by JobcenterTycoon** [comment]
Test

```
@@||googletagmanager.com/gtag/js?$script,domain=promo-visits.site|satoshi-win.xyz|shorterall.com
@@||google-analytics.com/analytics.js$script,domain=promo-visits.site|satoshi-win.xyz|shorterall.com
promo-visits.site,satoshi-win.xyz,shorterall.com#@#+js(no-window-open-if)
promo-visits.site,satoshi-win.xyz,shorterall.com##+js(rmnt, script, break;case $.)
promo-visits.site,satoshi-win.xyz,shorterall.com###outer
promo-visits.site,satoshi-win.xyz,shorterall.com##div[style^="position: fixed; display: block; width:"]
||missena.io^$3p
||promo-visits.site^$popup
```
